libevm/JitVM.cpp

```diff
@@ -248,47 +242,112 @@ int64_t evm_call(
 	return cost;
 }
 
+
+/// RAII wrapper for an evm instance.
+class EVM
+{
+	evm_instance* m_instance = nullptr;
+
+public:
+	EVM(evm_query_fn _queryFn, evm_update_fn _updateFn, evm_call_fn _callFn)
+	{
+		m_instance = evm_create(_queryFn, _updateFn, _callFn);
```

Comment:
Right, it does not matter much for the pointer, but still I prefer the other version because it is cleaner. As you would write
```
EVM(evm_query_fn _queryFn, evm_update_fn _updateFn, evm_call_fn _callFn):
    m_instance = evm_create(_queryFn, _updateFn, _callFn)
{
}
```

I would say it stands out in the same way.